.github/workflows/triage-issues.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           days-before-stale: 21
           days-before-close: 7
+          ignore-updates: true
```

Comment:
Reviewers please note: [ignore-issue-updates](https://github.com/actions/stale?tab=readme-ov-file#ignore-issue-updates) may be preferred.